hedera-node/hedera-schedule-service-impl/src/main/java/com/hedera/node/app/service/schedule/impl/handlers/ScheduleCreateHandler.java

```diff
@@ -238,11 +270,29 @@ private void checkSchedulableWhitelistHandle(final Schedule provisionalSchedule,
         }
     }
 
+    private void validatePayerAndScheduler(
+            final PreHandleContext context, final ScheduleCreateTransactionBody scheduleBody) throws PreCheckException {
+        final ReadableAccountStore accountStore = context.createStore(ReadableAccountStore.class);
+        final AccountID payerForSchedule = scheduleBody.payerAccountID();
+        if (payerForSchedule != null) {
+            final Account payer = accountStore.getAccountById(payerForSchedule);
+            if (payer == null) {
```

Comment:
Here we just need to check that the payer exists.  Currently, we should not check more until execution, because mono assumes nearly all checks wait until then.
In the future, after differential testing is completed, we should definitely consider making this check more robust.